ARCHITECTURE.md

```diff
@@ -45,27 +48,27 @@ Kedro-Viz loads data asynchronously in production from the API, or when using th
 
 Each pipeline endpoint corresponds to a different top-level pipeline. Only one top-level pipeline should be loaded at a time, so loading data from a pipeline endpoint will reset the pipeline state in the store. Each pipeline dataset contains all the data required to render the graph.
 
-On first page-load, the app always loads the `/api/main` endpoint first. This is the endpoint that corresponds to the 'default' pipeline. The app can load other pipelines from `/api/pipeline/<id>`. If another pipeline is saved as the active pipeline in localStorage, and if it exists in the current project, then the app will load that pipeline on first page-load, but it will always load the `/api/main` endpoint first, in order to check whether the active pipeline is present at that endpoint before requesting it.
+On first page-load, the app always loads the `/api/main` endpoint first. This is the endpoint that corresponds to the 'default' pipeline. The app can load other pipelines from `/api/pipeline/<id>`. If another pipeline is saved as the active pipeline in local storage, and if it exists in the current project, then the app will load that pipeline on first page load, but it will always load the `/api/main` endpoint first, in order to check whether the active pipeline is present at that endpoint before requesting it.
```

Comment:
the localStorage we refer to here is the `localstorage` mechanism under `window.localStorage` - I worry that calling it local storage instead of the specific name might cause confusion? 

We could re-word this to either 'Browser local storage' or as 'the `localStorage` object'?

Reply:
I agree, this should be kept as is - if necessary we can clarify it by wrapping it in backticks to show that we mean the object: i.e. `localStorage` not localStorage